09_2024_scikit_learn_priorities.md

```diff
@@ -0,0 +1,260 @@
+# Open source software priorities at Probabl
+
+At Probabl, we continue our dedicated efforts to support and enhance `scikit-learn` and
+its ecosystem. In this post, we provide a retrospective on the work we have
+accomplished on `scikit-learn` and other supported open-source packages over the past
```

Comment:
This gives the impression that the list below was accomplished by probabl alone.

Reply:
I end up with a section to exactly addressed such impression. So I should move right at the beginning such that it is clear that this is not the case.

Reply:
I moved the acknowledgement right at the start.